crates/ruff/src/rules/flake8_bugbear/rules/mutable_argument_default.rs

```diff
@@ -50,14 +53,23 @@ use crate::checkers::ast::Checker;
 pub struct MutableArgumentDefault;
 
 impl Violation for MutableArgumentDefault {
+    const AUTOFIX: AutofixKind = AutofixKind::Sometimes;
+
     #[derive_message_formats]
     fn message(&self) -> String {
         format!("Do not use mutable data structures for argument defaults")
     }
+    fn autofix_title(&self) -> Option<String> {
+        Some(format!("Replace mutable data structure with `None` in argument default and replace it with data structure inside the function if still `None`"))
+    }
```

Comment:
Maybe swap the title and message? The title should be short to fit into the IDE popup. The message is allowed to be more extensive.